pkg/event-manager/handlers_driver.go

```diff
@@ -0,0 +1,256 @@
+///////////////////////////////////////////////////////////////////////
+// Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+///////////////////////////////////////////////////////////////////////
+
+package eventmanager
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	apiclient "github.com/go-openapi/runtime/client"
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/swag"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/vmware/dispatch/pkg/entity-store"
+	"github.com/vmware/dispatch/pkg/event-manager/gen/models"
+	driverapi "github.com/vmware/dispatch/pkg/event-manager/gen/restapi/operations/drivers"
+	"github.com/vmware/dispatch/pkg/secret-store/gen/client/secret"
+	"github.com/vmware/dispatch/pkg/trace"
+	"github.com/vmware/dispatch/pkg/utils"
+)
+
+var builtInDrivers = map[string]map[string]bool{
```

Comment:
Do we really want to hard-code the built in drivers?  Is there a way we can load them up from a manifest?

Reply:
Well, there are built-in, so... What would be the benefit of using manifest here?